### DISCUSSION TITLE
Server-recorded project wav filenames show full IPv6 address

### DISCUSSION THREAD

**Discussion by rdica** [jamulussoftware/jamulus#2298] [General]
I recorded a jam last night and after downloading the project to my local DAW, I discovered the wav file generated by my clients connection displayed the whole IPv6 address of the client, unlike the filename of the client using legacy IP address:

```
ardy@host:~(0)$ ll Jam-20220126-234046629/
total 122472
drwxr-xr-x 2 ardy ardy     4096 Jan 27 14:44 ./
drwxrwxr-x 3 ardy ardy     4096 Jan 27 13:38 ../
-rw-r--r-- 1 ardy ardy 62696492 Jan 27 13:38 Hey_Joe_-75_163_73_x_22175-1-2.wav
-rw-r--r-- 1 ardy ardy      158 Jan 27 13:38 Jam-20220126-234046629.lof
-rw-r--r-- 1 ardy ardy     5617 Jan 27 14:44 Jam-20220126-234046629.rpp
-rw-r--r-- 1 ardy ardy 62696492 Jan 27 13:38 RickD-2001_1055_8000_df70_1ad6_c7ff_fe01_a562_x_22217-0-2.wav
```
As shown above, the legacy IP has it's last octet masked but on the IPv6 address all 8 nybbles are shown and the `_x_` is masking what?

Is this a bug?

**Comment by ghost** [comment]
I suggest that No part of an ip-address be used.
Instead use:
Jam-unix_time.wav or Jam-increment_number.wav
Maybe allow Jam to be changeable.

**Comment by pljones** [comment]
> As shown above, the legacy IP has it's last octet masked but on the IPv6 address all 8 nybbles are shown and the `_x_` is masking what?

I guess the IPv6 support failed to change that correctly.

**Reply by rdica** [reply]
Right, so is this a bug?

**Reply by ann0see** [reply]
Yes

**Reply by rdica** [reply]
@ann0see Do I need to move this to Bug Investigation or will the team take care of this and onwards?

**Reply by ann0see** [reply]
Please open an issue for this bug

**Reply by rdica** [reply]
https://github.com/jamulussoftware/jamulus/issues/2342 opened, thanks.
